__tests__/index.test.js

```diff
@@ -7,17 +7,9 @@ describe("Purchases", () => {
     jest.resetAllMocks();
   });
 
-  it("isUTCDateStringFuture returns true when a date is in the future", () => {
-    const {isUTCDateStringFuture} = require("../index");
-    const dateAhead = new Date();
-    dateAhead.setDate(dateAhead.getDate() + 2);
-
-    expect(isUTCDateStringFuture(dateAhead.toUTCString())).toEqual(true);
-  });
```

Comment:
`isUTCDateStringFuture` was only used in tests, is there a reason to keep it? @vegaro